Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -178,8 +178,15 @@ public void Process(ITelemetry telemetry)
             }
         }
 
-        private static ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelemetry requestTelemetry)
+        private static ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelemetry requestTelemetry, IApplicationIdProvider applicationIdProvider)
         {
+#if NET45
+            // some of the requestTelemetry properties might be deffered by using RequestTrackingTelemetryModule.DisableTrackingProperties.
+            // evaluate them now:
+            var request = System.Web.HttpContext.Current?.Request;
```

Comment:
Can `System.Web.HttpContext.Current` throw when not in ASP.NET context? If so - please add try/catch so console apps wouldn't blow out. I'd suggest to do it anyway as I believe this call is protected with some security demands which may not be met on some console apps

Reply:
Good point, fixed